cluster/gce/util.sh

```diff
@@ -1909,18 +1921,28 @@ function make-gcloud-network-argument() {
     fi
     ret="${ret},subnet=${subnetURL}"
     ret="${ret},aliases=pods-default:${alias_size}"
+    # if ${internal_address} is specified, attach it as an /32 alias so it's possible to
+    # detach it later.
+    if [[ -n ${internal_address:-} ]]; then
+      ret="${ret};${internal_address}"
+    fi
     ret="${ret} --no-can-ip-forward"
   else
+    ret="--network-interface"
```

Comment:
It seems we're setting it in both if and else - can you move it outside if then?